tests/TalkTest.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace App\Tests\Entity;
+
+use App\Entity\Submit;
+use App\Entity\Talk;
+use PHPUnit\Framework\TestCase;
+
+class TalkTest extends TestCase
```

Comment:
This file is not in the right directory. It should follow the same convention as the `src/` dir